clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -82,14 +107,8 @@ const char *SYCL::Linker::constructLLVMLinkCommand(Compilation &C,
   TranslateSYCLLinkerArgs(C, Args, getToolChain(), CmdArgs);
   // Add an intermediate output file.
   CmdArgs.push_back("-o");
-  const char *OutputFileName = nullptr;
-  if (ToBc) {
-    SmallString<128> TmpName(C.getDriver().GetTemporaryPath(
-                             OutputFilePrefix.str() + "-linked", "bc"));
-    OutputFileName = C.addTempFile(C.getArgs().MakeArgString(TmpName));
-    CmdArgs.push_back(OutputFileName);
-  } else
-    CmdArgs.push_back(Output.getFilename());
```

Comment:
As ToBc has been removed, there is no need for the parameter anymore in the constructLLVMCommand function, so it can be cleaned up.

Reply:
Removed.